### PR TITLE
refactor(hosting.private.database.state): use oui-tile

### PR DESCRIPTION
**Pull request by TedKac**
## refactor(hosting.private.database.state): use oui-tile

ref: MBP-424

### CODE REVIEW

File: client/app/private-database/state/private-database-state.html [hunk 1]
```diff
@@ -6,382 +6,276 @@
...
-                                                    <span class="ml-auto my-auto"
-                                                          data-ng-show="taskState.changeFtpPassword">
-                                                        <oui-spinner data-size="s"></oui-spinner>
-                                                        <span data-translate="privateDatabase_action_changepassword_doing"></span>
-                                                    </span>
-                                                </div>
-                                            </div>
-                                        </div>
-                                    </li>
-                                    <li class="oui-tile__item"
```
Comment (TedKac) [inline client/app/private-database/state/private-database-state.html]: I took the opportunity to remove the legacy part that is no longer used.

File: client/app/private-database/state/private-database-state.html [hunk 2]
```diff
@@ -6,382 +6,272 @@
...
+                                    <span data-ng-if="stateCtrl.database.infrastructure === 'docker'">admin</span>
+                                </oui-tile-description>
+                            </oui-tile-definition>
+                            <oui-tile-definition data-term="{{ ::'privateDatabase_password_' + stateCtrl.database.infrastructure | translate}}">
+                                <oui-tile-description>
+                                    <span data-ng-show="!taskState.changeFtpPassword">********</span>
+                                    <div class="ml-auto my-auto"
+                                         data-ng-if="!stateCtrl.isExpired">
+                                        <span class="ml-auto my-auto"
+                                              data-ng-show="taskState.changeFtpPassword">
```
Comment (FredericEspiau) [inline client/app/private-database/state/private-database-state.html]:
```suggestion
                                              data-ng-if="taskState.changeFtpPassword">
```

File: client/app/private-database/state/private-database-state.html [hunk 3]
```diff
@@ -6,382 +6,272 @@
...
+                                   </button>
+                                </oui-tile-description>
+                                <oui-action-menu data-compact data-align="end"
+                                                 data-ng-if="!stateCtrl.isExpired">
+                                    <oui-action-menu-item data-on-click="setAction('database/ram/update/private-database-database-ram-update', stateCtrl.database)"
+                                                          data-ng-disabled="taskState.lockAction">
+                                        <span data-translate="privateDatabase_change_ram_title"></span>
+                                    </oui-action-menu-item>
+                                    <oui-action-menu-item data-on-click="stateCtrl.goToOom()"
+                                                          data-ng-disabled="taskState.lockAction">
```
Comment (marie-j) [inline client/app/private-database/state/private-database-state.html]:
```suggestion
                                                          data-disabled="taskState.lockAction">
```
According to documentation

File: client/app/private-database/state/private-database-state.html [hunk 4]
```diff
@@ -6,382 +6,272 @@
...
+                                       <span data-ng-if="stateCtrl.database.oom.realList.length === 0" class="fa fa-check-circle" aria-hidden="true"></span>
+                                       <span data-ng-if="stateCtrl.database.oom.realList.length > 0" data-ng-bind-html="'privateDatabase_warning_oom' | translate: { t0: stateCtrl.database.oom.realList.length }"></span>
+                                       <span data-ng-if="stateCtrl.database.oom.realList.length === 0" data-ng-bind-html="'privateDatabase_no_oom' | translate"></span>
+                                   </button>
+                                </oui-tile-description>
+                                <oui-action-menu data-compact data-align="end"
+                                                 data-ng-if="!stateCtrl.isExpired">
+                                    <oui-action-menu-item data-on-click="setAction('database/ram/update/private-database-database-ram-update', stateCtrl.database)"
+                                                          data-ng-disabled="taskState.lockAction">
```
Comment (marie-j) [inline client/app/private-database/state/private-database-state.html]:
```suggestion
                                                          data-disabled="taskState.lockAction">
```

File: client/app/private-database/state/private-database-state.html [hunk 5]
```diff
@@ -6,382 +6,272 @@
...
+                                    </ul>
+                                </oui-tile-description>
+                            </oui-tile-definition>
+                        </oui-tile>
+                        <oui-tile data-heading="{{ ::'privateDatabase_dashboard_admin_informations' | translate}}"
+                                  data-ng-if="stateCtrl.database.offer === 'classic'"
+                                  class="mb-5">
+                            <oui-tile-definition data-term="{{ ::'privateDatabase_gui' | translate}}">
+                                <oui-tile-description>
+                                    <a data-ng-href="{{stateCtrl.database.guiURL}}" target="_blank"
```
Comment (marie-j) [inline client/app/private-database/state/private-database-state.html]:
```suggestion
                                    <a data-ng-href="{{stateCtrl.database.guiURL}}" target="_blank"
                                    rel="noopener"
```

File: client/app/private-database/state/private-database-state.html [hunk 6]
```diff
@@ -6,382 +6,272 @@
...
+                            <h4 class="oui-header_5 oui-tile__definition"
+                                data-ng-bind="'privateDatabase_dashboard_ftp_informations_' + stateCtrl.database.infrastructure | translate"></h4>
+                            <oui-tile-definition data-term="{{ ::'privateDatabase_dashboard_ftp_username' | translate}}">
+                                <oui-tile-description>
+                                    <span data-ng-if="stateCtrl.database.infrastructure === 'docker'">admin</span>
+                                </oui-tile-description>
+                            </oui-tile-definition>
+                            <oui-tile-definition data-term="{{ ::'privateDatabase_password_' + stateCtrl.database.infrastructure | translate}}">
+                                <oui-tile-description>
+                                    <span data-ng-show="!taskState.changeFtpPassword">********</span>
```
Comment (marie-j) [inline client/app/private-database/state/private-database-state.html]:
```suggestion
                                    <span data-ng-if="!taskState.changeFtpPassword">********</span>
```

File: client/app/private-database/state/private-database-state.html [hunk 7]
```diff
@@ -6,382 +6,272 @@
...
+                                              data-ng-show="taskState.changeFtpPassword">
+                                            <oui-spinner data-size="s"></oui-spinner>
+                                            <span data-translate="privateDatabase_action_changepassword_doing"></span>
+                                        </span>
+                                    </div>
+                                </oui-tile-description>
+                                <oui-action-menu data-compact data-align="end"
+                                                 data-ng-if="!taskState.changeFtpPassword">
+                                    <oui-action-menu-item data-on-click="stateCtrl.changeFtpPassword()"
+                                                          data-ng-disabled="taskState.lockAction">
```
Comment (marie-j) [inline client/app/private-database/state/private-database-state.html]:
```suggestion
                                                          data-disabled="taskState.lockAction">
```

**Pull request merged**